src/engine/strat_engine/pool.rs

```diff
@@ -359,13 +358,16 @@ impl StratPool {
 }
 
 impl Pool for StratPool {
+    /// Create a filesystem in the Pool.
```

Comment:
Please put the lines below not here, but on the comment for the Pool trait's ```create_filesystem``` method in ```src/engine/engine.rs". Please remove the line above.